crates/sui-framework/packages/move-stdlib/sources/string.move

```diff
@@ -69,33 +80,44 @@ module std::string {
         s.append(utf8(bytes))
     }
 
-    /// Insert the other string at the byte index in given string. The index must be at a valid utf8 char
-    /// boundary.
+    /// Insert the other string at the byte index in given string. The index
+    /// must be at a valid utf8 char boundary.
     public fun insert(s: &mut String, at: u64, o: String) {
         let bytes = &s.bytes;
-        assert!(at <= bytes.length() && internal_is_char_boundary(bytes, at), EINVALID_INDEX);
+        assert!(
+            at <= bytes.length() && internal_is_char_boundary(bytes, at),
+            EInvalidIndex
+        );
         let l = s.length();
-        let mut front = s.sub_string(0, at);
-        let end = s.sub_string(at, l);
+        let mut front = s.substring(0, at);
+        let end = s.substring(at, l);
         front.append(o);
         front.append(end);
         *s = front;
     }
 
-    /// Returns a sub-string using the given byte indices, where `i` is the first byte position and `j` is the start
-    /// of the first byte not included (or the length of the string). The indices must be at valid utf8 char boundaries,
+    /// Alias to align the API with `ASCII` and standardize naming.
+    public use fun sub_string as String.substring;
```

Comment:
I would have a similar question here as Todd above on whether this should be a method alias or a function that `sub_string` forwards too.

Reply:
Commented in the thread above, but yes, let's do a separate function to simplify deprecating.